packages/core/admin/admin/src/components/MainNav/NavUser.tsx

```diff
@@ -67,7 +72,7 @@ export const NavUser = ({ children, initials, ...props }: NavUserProps) => {
             })}
           </MenuItem>
 
-          <MenuItem onSelect={handleLogout} color="danger600">
+          <MenuItem onSelect={handleLogout} color="danger600" $variant="danger">
```

Comment:
I think you can simplify your solution just extending the MenuItem styled component and adding the danger background

something like that

```
const MenuItem = styled(Menu.Item)`
  & > span {
    width: 100%;
    display: flex;
    align-items: center;
    gap: ${({ theme }) => theme.spaces[3]};
    justify-content: space-between;
  }
`;

const MenuItemDanger = styled(MenuItem)`
&hover {
  background: theme.colors.danger100;
  }
`
```